cmd/tlsx/main.go

```diff
@@ -105,6 +105,7 @@ func readFlags() error {
 		flagSet.IntVarP(&options.Concurrency, "concurrency", "c", 300, "number of concurrent threads to process"),
 		flagSet.IntVar(&options.Timeout, "timeout", 5, "tls connection timeout in seconds"),
 		flagSet.IntVar(&options.Retries, "retries", 3, "number of retries to perform for failures"),
+		flagSet.IntVar(&options.Delay, "delay", 0, "delay between each connection in milliseconds per thread, use concurrency to fine tune the absolute number of connections made"),
```

Comment:
instead of harcoding duration as ms, we should instead parse a human friendly duration format. ex. `200ms`, `1s`, etc.